TedTalks/Managers/TedTalkManager.swift

```diff
@@ -10,12 +10,41 @@ import Foundation
 class TedTalkManager {
     
     private(set) var tedTalks: [TedTalk] = []
+    private(set) var filteredTedTalks: [TedTalk] = []
     
     init() {
         self.tedTalks = Serialization.load(fromJSON: "ted") ?? []
+        self.filteredTedTalks = self.tedTalks
     }
     
     func add(tedTalk: TedTalk) {
         tedTalks.append(tedTalk)
     }
+    
+    func filter(keyword: String, filter: String) {
+        filteredTedTalks = keyword=="" ? tedTalks.filter{ doesTedTalkMatch(keyword, filter, $0)} : tedTalks
```

Comment:
Let's always add spaces to the left and right of operands, in this case `keyword == ""`

Reply:
Fixed